compiler-plugin/src/main/kotlin/com/google/devtools/ksp/KotlinSymbolProcessingExtension.kt

```diff
@@ -134,14 +144,15 @@ abstract class AbstractKotlinSymbolProcessingExtension(val options: KspOptions,
                     anyChangesWildcard,
                     ksFiles
             )
-            processors.forEach init@{ processor ->
+            processors = providers.mapNotNull { provider ->
```

Comment:
This changes the behavior a bit, because `onError` and `finish` won't be called if any exception is thrown while initializing the processor. I guess it makes sense since there's not much the processor can do if it can't even be initialized.